proto/semantic.proto

```diff
@@ -174,3 +178,30 @@ message Span {
   Position start = 1;
   Position end = 2;
 }
+
+message ScopeGraphFile {
+  string path = 1;
+  string language = 2;
+  repeated ScopeGraphNode nodes = 3;
+  repeated ScopeGraphPath paths = 4;
+  repeated ParseError errors = 5;
+}
+
+message ScopeGraphNode {
+  int32 id = 1;
+  string name = 2;
+  string line = 3;
+  string syntax = 4;
```

Comment:
What does `syntax` mean in this context? Does this describe the node type (push, pop, etc?) If so I imagine we’ll do some parsing when converting to the native data type.

Reply:
I'm going to rename this to `kind` (which is what we call this in the other proto responses) - It's essential the name of the syntax term (`Function`, `Call`, etc).